common.tf

```diff
@@ -366,7 +366,13 @@ module "cloudwatch_alarm_event_bridge_rule" {
       module.dr2_custodial_copy_db_builder_queue.dlq_cloudwatch_message_visible_alarm_arn,
       module.dr2_custodial_copy_notifications_queue.dlq_cloudwatch_message_visible_alarm_arn,
       module.dr2_external_notifications_queue.dlq_cloudwatch_message_visible_alarm_arn,
-      module.dr2_copy_files_from_tdr_sqs.dlq_cloudwatch_message_visible_alarm_arn
+      module.dr2_copy_files_from_tdr_sqs.dlq_cloudwatch_message_visible_alarm_arn,
```

Comment:
I know you didn't create the original list but would you mind grouping these so that it's easier to see if we've missed something? e.g.

```
module.dr2_copy_files_from_tdr_sqs.queue_cloudwatch_message_visible_alarm_arn,
module.dr2_copy_files_from_tdr_sqs.dlq_cloudwatch_message_visible_alarm_arn
```